src/main/java/Duke.java

```diff
@@ -1,10 +1,68 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    static final String LOGO = "$$$$$$$\\                  $$\\           \n" +
+            "$$  __$$\\                 $$ |          \n" +
+            "$$ |  $$ |$$\\   $$\\  $$$$$$$ | $$$$$$\\  \n" +
+            "$$ |  $$ |$$ |  $$ |$$  __$$ |$$  __$$\\ \n" +
+            "$$ |  $$ |$$ |  $$ |$$ /  $$ |$$$$$$$$ |\n" +
+            "$$ |  $$ |$$ |  $$ |$$ |  $$ |$$   ____|\n" +
+            "$$$$$$$  |\\$$$$$$  |\\$$$$$$$ |\\$$$$$$$\\ \n" +
+            "\\_______/  \\______/  \\_______| \\_______|";
+    static final String GREET = "I am Dude.\n" +
+            "Type something in, dude.";
+    static final String BYE = "Catch you later, dude.";
+
+    static List<Task> tasks = new ArrayList<>();
+
+    public static void runCommand(String command) {
+        String[] args = command.split(" ");
+        switch (args[0]) {
+        case "bye":
+            System.out.println(BYE);
+            System.exit(0);
+            // no break needed, the code has already exited
+        case "list":
+            for (int i = 0; i < tasks.size(); i++) {
+                Task task = tasks.get(i);
+                System.out.printf("%d.[%s] %s\n", i + 1, (task.isDone() ? "X" : " "), task.getDescription());
```

Comment:
Perhaps more could be done to make this line more readable? Consider not using the lambda or putting it in a different line.

Reply:
I agree, it would be great to make this line more understandable